config/src/test/java/org/springframework/security/config/annotation/web/configurers/saml2/Saml2LoginConfigurerTests.java

```diff
@@ -106,10 +106,10 @@
 public class Saml2LoginConfigurerTests {
 
 	private static final Converter<Assertion, Collection<? extends GrantedAuthority>> AUTHORITIES_EXTRACTOR = (
-			a) -> Arrays.asList(new SimpleGrantedAuthority("TEST"));
+			a) -> Collections.singletonList(new SimpleGrantedAuthority("TEST"));
```

Comment:
Unless these changes were required by the style checker, will you please place any style or other cleanup changes into a second commit that follows the feature commit? This will ensure fewer merge conflicts if the feature gets backported.